Makefile

```diff
@@ -396,3 +397,127 @@ image-mnist-job-test-build: ## Build container image with the MNIST job.
 .PHONY: image-mnist-job-test-push
 image-mnist-job-test-push: image-mnist-job-test-build ## Push container image with the MNIST job.
 	podman push ${MNIST_JOB_TEST_IMG}
+
+# RHOAI/ODH related resources installation
+
+##@ all-in-one
```

Comment:
Could we add a basic usage example as a comment. In case anyone encounters this not working on ODH.